packages/terra-alert/tests/jest/Alert.test.jsx

```diff
@@ -24,7 +24,7 @@ describe('Alert with no props', () => {
 
   it('throws error on missing locale prop in Base', () => {
     try {
-      shallow(<Alert />);
+      render(<Alert />);
```

Comment:
shallow doesn't pass a context to the component anymore, so we need to do a full render to test that condition.